tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "compilerOptions": {
     "outDir": "lib",
-    "target": "es5",
+    "target": "es6",
```

Comment:
What's your reason for dropping es5 support? Is that needed for publishing an ES Module?

We currently support es5 so as to support IE 11.